FogCreek/Glitch-Community
# Charmed albertonykus: create /settings route for user deletion

## Links
https://charmed-albertonykus.glitch.me/settings
[ch3219]

## GIF/Screenshots:
![Screen Shot 2019-10-28 at 2 58 42 PM](https://user-images.githubusercontent.com/3011231/67708957-792cd200-f993-11e9-8f9a-b0c5cd962fea.png)


## Changes:
* Moved password and 2fa settings into a page (instead of a modal)
* Changed avatar menu so that the account settings button links to that page

## How To Test:
* Turn on some feature switches
* Go to /settings
* Make sure what you see jives with the feature switches you have set
* Make sure you can use the features

## Feedback I'm looking for:
Should I remove the (disabled) privacy button?
The modal has a "Email notifications are sent to {email}" message, should we have that here too?
